extensions/scarb-cairo-run/src/deserialization.rs

```diff
@@ -91,8 +91,8 @@ impl Args {
         for arg in iterator {
             match arg {
                 Value::Number(n) => {
-                    let n = n.as_u64().ok_or(ArgsError::NumberOutOfRange)?;
-                    args.push(Arg::Value(Felt252::from(n)));
+                    let n = Felt252::from_str(n.to_string().as_str())?;
```

Comment:
nitpick: I winder if we could get rid of intermediate string creation (from_str -> to_string -> as_str)
maybe we could consider using format! ? I imageine something like this could work `Felt252::from_str(&format!("{}", n))` . I guess this could be more idiomatic, but am not strong on this (hence nitpick)

Reply:
I don't really see how would it be different, as format! does allocate a String anyway?

Reply:
shoot, you're right, I forgot. yeah, then this makes this comment even more of a nitpick, as this would probably be purely cosmetic change. completely up to you, both are fine for me :+1: